backend/src/controller/defenceController.ts

```diff
@@ -43,7 +42,9 @@ function handleDefenceActivation(req: DefenceActivateRequest, res: Response) {
 		return;
 	}
 
-	if (req.session.levelState[level].defences === undefined) {
+	const currentDefences = req.session.levelState[level].defences;
+
+	if (currentDefences === undefined) {
```

Comment:
`if (!currentDefences)` is a little more robust, as it can handle null or undefined.

Same further down.